fixture/hr_administration.py

```diff
@@ -0,0 +1,32 @@
+from fixture.step import StepHelper
+from selenium.webdriver.remote.webdriver import WebDriver
+
+class HrAdministration:
+    add_user_button = "//div[@id='systemUserDiv'] //*[text()='add']"
+    filter_users_button = '//a[@data-tooltip="Filter"]/i'
+    filtered_username = '//span[text()="admin"]'
+    filtered_user_role = '//span[text()="Global Admin"]'
+    filter_no_records_message = '//div[text()="No Records Found"]'
+
+    def __init__(self, step: StepHelper, wd: WebDriver):
+        self.step = step
+        self.wd = wd
+
+    def click_add_user(self):
+        self.step.click_on_element(self.add_user_button)
+
+    def click_filter_button(self):
+        self.step.click_on_element(self.filter_users_button)
+
+    def get_filtered_username(self):
+        return self.step.get_element_text(self.filtered_username)
```

Comment:
please use method 'get_elements_texts' from step, and change the method name to get_filtered_usernames as you will get the list of elements